client/packages/host/src/components/SyncProgress/SyncProgress.tsx

```diff
@@ -54,7 +54,14 @@ export const SyncProgress: FC<SyncProgressProps> = ({
                     )
                   }
                   error={isActiveAndError}
-                  optional={progress && `${progress.done}/${progress.total}`}
+                  optional={
+                    progress && (
+                      <Box
+                        display={'flex'}
+                        justifyContent="center"
+                      >{`${progress.done}/${progress.total}`}</Box>
```

Comment:
haha the little wizard is looking wicked 😁

Reply:
🙏 was thinking to extract this Box into method btw, but looked ok inline